frontend/packages/console-app/src/components/quick-starts/QuickStartEmptyState.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateActions,
+  EmptyStateFooter,
+  EmptyStateVariant,
+  Skeleton,
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons/dist/esm/icons/cubes-icon';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+import { WrenchIcon } from '@patternfly/react-icons/dist/esm/icons/wrench-icon';
+import { useTranslation } from 'react-i18next';
+import { QuickStartModel } from '@console/app/src/models';
+import { useAccessReview } from '@console/dynamic-plugin-sdk/src/app/components/utils/rbac';
+import { getReferenceForModel } from '@console/dynamic-plugin-sdk/src/utils/k8s';
+import { LinkTo } from '@console/shared/src/components/links/LinkTo';
+
+const QUICK_START_DOCS_URL =
```

Comment:
Move to https://github.com/openshift/console/blob/main/frontend/public/components/utils/documentation.tsx so all doc URLs are in the same file (the docs team monitors this).